src/linear_algebra/basic.lean

```diff
@@ -602,7 +602,7 @@ submodule.ext $ λ a, by simp
 lemma map_comp [ring_hom_surjective σ₂₃] [ring_hom_surjective σ₁₃]
   (f : M →ₛₗ[σ₁₂] M₂) (g : M₂ →ₛₗ[σ₂₃] M₃)
   (p : submodule R M) : map (g.comp f : M →ₛₗ[σ₁₃] M₃) p = map g (map f p) :=
-set_like.coe_injective $ by simp [map_coe]; rw ← image_comp
```

Comment:
Does `exact image_comp _ _` or similar close this goal?

Reply:
no, it does not, I've played around and I did not find a simple term proof.